src/Pos/Client/Txp/History.hs

```diff
@@ -118,9 +130,9 @@ getTxsByPredicate pr mDiff mTs txs = go txs []
   where
     go [] acc = return acc
     go ((wh@(WithHash tx txId), _wit, dist) : rest) acc = do
-        inputs <- getSenders tx
+        inputs <- mergeTxOuts <$> getSenders tx
```

Comment:
Waaait a minute, so if here transaction spends several outputs which belong to _one_ address, here we'll get that transaction has only one input resolved? But this simply _isn't true_ -- we should display _real_ outputs.